Hudl.Mjolnir.Tests/Bulkhead/SemaphoreBulkheadHolderTests.cs

```diff
@@ -136,41 +133,41 @@ public void Construct_WhenMaxConcurrentConfigIsInvalid_DoesSomething()
             Assert.Equal(invalidMaxConcurrent, exception.ActualValue);
         }
 
-        [Fact]
-        public void UpdateMaxConcurrent_IgnoresInvalidValues()
-        {
-            // Arrange
-
-            var key = AnyGroupKey;
-            const int invalidMaxConcurrent = -1;
-
-            var mockMetricEvents = new Mock<IMetricEvents>(MockBehavior.Strict);
-            mockMetricEvents.Setup(m => m.BulkheadGauge(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<int>(), It.IsAny<int>()));
-
-            var mockBulkheadConfig = new Mock<IBulkheadConfig>(MockBehavior.Strict);
-            mockBulkheadConfig.Setup(m => m.GetMaxConcurrent(key)).Returns(AnyPositiveInt);
-            mockBulkheadConfig.Setup(m => m.AddChangeHandler(key, It.IsAny<Action<int>>()));
-            mockBulkheadConfig.Setup(m => m.GetConfigKey(key)).Returns(key.Name);
-
-            var mockLog = new Mock<IMjolnirLog>(MockBehavior.Strict);
-            mockLog.Setup(m => m.Error(It.IsAny<string>()));
-
-            var mockLogFactory = new Mock<IMjolnirLogFactory>(MockBehavior.Strict);
-            mockLogFactory.Setup(m => m.CreateLog(It.IsAny<Type>())).Returns(mockLog.Object);
-
-            var holder = new SemaphoreBulkheadHolder(key, mockMetricEvents.Object, mockBulkheadConfig.Object, mockLogFactory.Object);
-
-            // Act
-
-            var initialBulkhead = holder.Bulkhead;
-            holder.UpdateMaxConcurrent(invalidMaxConcurrent);
-
-            // Assert
-
-            // Bulkhead should be unchanged.
-            Assert.True(initialBulkhead == holder.Bulkhead);
-            mockLog.Verify(m => m.Error($"Semaphore bulkhead config {key.Name} changed to an invalid limit of {invalidMaxConcurrent}, the bulkhead will not be changed"), Times.Once);
-        }
+//        [Fact]
```

Comment:
Same here - reimplemented?

Reply:
This has been done in https://github.com/hudl/Mjolnir/pull/64